packaging/pkg_helpers.bash

```diff
@@ -303,3 +303,38 @@ setup_junit_results_folder() {
     export CONDA_PYTORCH_BUILD_RESULTS_DIRECTORY="${SOURCE_ROOT_DIR}/build_results/results.xml"
   fi
 }
+
+
+download_copy_ffmpeg() {
+  mkdir ffmpeg_tmp
+  cd ffmpeg_tmp
+  if [[ "$OSTYPE" == "msys" ]]; then
+    conda install -yq ffmpeg -c pytorch
```

Comment:
It is a bit weird that you pinned the version of `ffmpeg` in the following line but not in this one.

Reply:
I think we can remove the version pinning safely